zoo/policies/interaction_aware_motion_prediction/setup.cfg

```diff
@@ -0,0 +1,19 @@
+[metadata]
+name = smarts_agent_iamp_stub
```

Comment:
Is this name intended?

Reply:
This setup file simply installs `smarts_agent_iamp` package as dependency which contains the entire trained model. Thus I named this as `<package_name>_stub`. Any other naming system is welcomed. Alternatively, we may also consider sidestepping this setup file and somehow directly pip install `smarts_agent_iamp` package when users execute `scl zoo install <agent_locator>`.

Reply:
It seems fine for now.